R/methods-Links.R

```diff
@@ -0,0 +1,32 @@
+# Methods for Link and LinkList objects
+
+# Accessors for fanciness
```

Comment:
I kinda like get* and set*..

Reply:
is there a particular reason? If it's a preference i'd rather stick to what we have because it's more in line with what i'm guessing is more standard R practice since it follows Hadley's book.

Reply:
It's a preference, and not a particularly strong one at that. I've done it myself both ways before, but I've started recently to feel get* and set* is a bit more readable is all.